opts/address_pools.go

```diff
@@ -0,0 +1,80 @@
+package opts
+
+import (
+	"encoding/csv"
+	"fmt"
+	"strconv"
+	"strings"
+
+	types "github.com/docker/libnetwork/ipamutils"
+)
+
+// PoolsOpt is a Value type for parsing the default address pools definitions
+type PoolsOpt struct {
+	values *[]*types.NetworkToSplit
```

Comment:
I don't think this needs to be a pointer. The `PoolsOpt` receiver is already a pointer, so you should be able to `p.values = append(p.values, ...)`